react-native-libraries.json

```
@@ -9555,5 +9555,14 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/ahmedrowaihi/react-native-i18n-storage",
+    "npmPkg": "@ahmedrowaihi/react-native-i18n-storage"
```

```suggestion
    "npmPkg": "@ahmedrowaihi/react-native-i18n-storage",
```

Whoops, missed a coma.